docs/gaia/join-testnet.md

```diff
@@ -103,6 +103,37 @@ curl https://raw.githubusercontent.com/cosmos/testnets/master/latest/genesis.jso
 Note we use the `latest` directory in the [testnets repo](https://github.com/cosmos/testnets)
 which contains details for the latest testnet. If you are connecting to a different testnet, ensure you get the right files.
 
+You now need to generate a `genesis tx` to generate a signed self-delegation:
+
+```bash
+gaiad gentx \
+    --amount <amount_of_delegation> \
+    --commission-rate <commission_rate> \
+    --commission-max-rate <commission_max_rate> \
+    --commission-max-change-rate <commission_max_change_rate> \
+    --pubkey <consensus_pubkey> \
+    --name <key_name>
+```
+
+If unspecified, `consensus_pubkey` will default to the output of `gaiad tendermint show-validator`.
+`key_name` is the name of the private key that will be used to sign the transaction.
+
+Consult `gaiad gentx --help` for more information on the flags defaults.
+
+Once you've collected all genesis transactions in `~/.gaiad/config/gentx`, you can run:
+
+```bash
+gaiad collect-gentxs
```

Comment:
When I run this I get the following error: 

```
$ gaiad collect-gentxs
ERROR: account ?+8{ϋ??dv?B???? not in genesis.json: map[]
```

Reply:
You need to run `add-genesis-account`. Such step should be taken by who is in charge of initialising genesis accounts